src/templates/components/PaidForHeader.svelte

```diff
@@ -14,44 +14,49 @@
 	export let SeriesUrl: GAMVariable;
 	export let ComponentTitle: GAMVariable;
 	export let edition: Single['branding']['edition'];
+	export let templateType = '';
 
 	export let popup = false;
 </script>
 
-<header>
-	<div class="paid">
-		<strong>Paid content</strong>
-		<Button
-			aria-controls="popup"
-			aria-expanded="true"
-			on:click={() => (popup = !popup)}
-			>About <ArrowDown width={12} flip={popup} /></Button
+<header class="{templateType}-header">
+	<div class="{templateType}-paid-content-and-title">
+		<div class="paid {templateType}-paid-content">
+			<strong>Paid content</strong>
+			<Button
+				aria-controls="popup"
+				aria-expanded="true"
+				on:click={() => (popup = !popup)}
+				>About <ArrowDown width={12} flip={popup} /></Button
+			>
+			{#if popup}
+				<div id="popup">
+					<p>
+						Paid content is paid for and controlled by an advertiser and
+						produced by the Guardian Labs team
+					</p>
+					<a
+						class="popup-link"
+						href={`${CLICK_MACRO}https://theguardian.com/content-funding`}
+						target="_top">Learn more about Guardian Labs content &rarr;</a
+					>
+				</div>
+			{/if}
+		</div>
+
+		<h1
+			class="adverts__title {templateType === 'multiple' && 'multiple-title'}"
```

Comment:
In svelte can do stuff like `class:multiple-title={templateType === 'multiple'}` which is a bit nicer.